File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/hero/HeroActivity.java
package com.jdroid.android.sample.ui.hero;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class HeroActivity extends FragmentContainerActivity {

	/**
	 * @see FragmentContainerActivity#getFragmentClass()
	 */
	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return HeroFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/hero/HeroActivity.kt
package com.jdroid.android.sample.ui.hero

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class HeroActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return HeroFragment::class.java
    }
}